!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  block_size: 21
  disp12_max_diff: 1
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      block_size: 21
      disp12_max_diff: 1
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      min_disparity: 0
      name: Default
      num_disparities: 128
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      prefilter_cap: 61
      prefilter_size: 5
      prefilter_type: 1
      smaller_block_size: 1
      speckle_range: 5
      speckle_window_size: 250
      state: true
      texture_threshold: 501
      type: ''
      uniqueness_ratio: 0
    state: []
  min_disparity: 0
  num_disparities: 128
  prefilter_cap: 61
  prefilter_size: 5
  prefilter_type: 1
  smaller_block_size: 1
  speckle_range: 5
  speckle_window_size: 250
  texture_threshold: 501
  uniqueness_ratio: 0
state: []
